Update/kage_011.txt

```diff
@@ -521,13 +521,13 @@ void main()
 //rあの心のうちに熱情を秘めるような人が、歳を取ってからの子どもを喜ばないはずがない。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "あの心のうちに熱情を秘めるような人が、歳を取ってからの子どもを喜ばないはずがない。",
-			NULL, "A person with passion in their heart won't lose the pleasure of raising children just because they've grown old.", Line_Normal);
+			NULL, "A person with passion in their heart like him would never not be happy to have a child just because they've grown old.", Line_Normal);
```

Comment:
```suggestion
			NULL, "A person with passion in their heart like him would never be unhappy to have a child just because they've grown old.", Line_Normal);
```